packages/react/src/generators/remote/files/module-federation/webpack.config.js__tmpl__

```diff
@@ -1,6 +1,6 @@
 const { composePlugins, withNx } = require('@nrwl/webpack');
```

Comment:
it's unable to find this package also please add it in devDependencies.
I could see this version is not stabilized